[2023-04-27 12:26:16,339] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.attack_ZooAttack manual__2023-04-27T12:25:48.345057+00:00 [queued]>
[2023-04-27 12:26:16,443] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.attack_ZooAttack manual__2023-04-27T12:25:48.345057+00:00 [queued]>
[2023-04-27 12:26:16,444] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-27 12:26:16,446] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-04-27 12:26:16,447] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-27 12:26:16,576] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): attack_ZooAttack> on 2023-04-27 12:25:48.345057+00:00
[2023-04-27 12:26:16,617] {standard_task_runner.py:52} INFO - Started process 3976 to run task
[2023-04-27 12:26:16,669] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_attack_dag', 'attack_ZooAttack', 'manual__2023-04-27T12:25:48.345057+00:00', '--job-id', '6834', '--raw', '--subdir', 'DAGS_FOLDER/attack_dag.py', '--cfg-path', '/tmp/tmpv41kax9g', '--error-file', '/tmp/tmpvm3a952p']
[2023-04-27 12:26:16,685] {standard_task_runner.py:80} INFO - Job 6834: Subtask attack_ZooAttack
[2023-04-27 12:26:17,270] {task_command.py:369} INFO - Running <TaskInstance: multi_attack_dag.attack_ZooAttack manual__2023-04-27T12:25:48.345057+00:00 [running]> on host e7e31d5a6603
[2023-04-27 12:26:17,898] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_attack_dag
AIRFLOW_CTX_TASK_ID=attack_ZooAttack
AIRFLOW_CTX_EXECUTION_DATE=2023-04-27T12:25:48.345057+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-27T12:25:48.345057+00:00
[2023-04-27 12:26:17,913] {transport.py:157} INFO - Attempting refresh to obtain initial access_token
[2023-04-27 12:26:17,923] {client.py:777} INFO - Refreshing access_token
[2023-04-27 12:26:18,813] {transport.py:157} INFO - Attempting refresh to obtain initial access_token
[2023-04-27 12:26:18,856] {client.py:777} INFO - Refreshing access_token
[2023-04-27 12:26:51,710] {pytorch.py:1198} INFO - Inferred 5 hidden layers on PyTorch classifier.
[2023-04-27 12:31:50,392] {attack.py:33} INFO - Optimizing...
[2023-04-27 12:31:50,430] {logging_mixin.py:115} INFO - I am right here on <class 'art.attacks.evasion.zoo.ZooAttack'>
[2023-04-27 12:31:50,433] {attack.py:39} INFO - Optimizing failed! using default: 0.04175758361816406
[2023-04-27 12:31:51,125] {python.py:173} INFO - Done. Returned value was: None
[2023-04-27 12:31:51,600] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_attack_dag, task_id=attack_ZooAttack, execution_date=20230427T122548, start_date=20230427T122616, end_date=20230427T123151
[2023-04-27 12:31:52,801] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-27 12:31:53,898] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
